samples/crypto/ecjpake/README.rst

```diff
@@ -0,0 +1,41 @@
+.. _crypto_ecjpake:
+
+Crypto: EC J-PAKE
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The EC J-PAKE sample demonstrates how to do password-authenticated key exchange.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+The sample performs the following operations:
+
+1. Initialization of the Platform Security Architecture (PSA) API.
```

Comment:
To be in line with the other steps, change to: Initializes the Platform Security Architecture (PSA) API.